sql/wsrep_binlog.cc

```diff
@@ -16,6 +16,7 @@
 #include "wsrep_binlog.h"
 #include "wsrep_priv.h"
 #include "log_event.h"
+#include "sql_base.h"
```

Comment:
It would be better to add why this file was included. A convention which is used in Server code.
```suggestion
#include "sql_base.h"  // TEMP_PREFIX
```

Reply:
done